lib/consoletest.pm

```diff
@@ -16,5 +16,11 @@ sub post_run_hook {
     $self->clear_and_verify_console;
 }
 
+sub post_fail_hook {
+    # Export logs after failure
+    script_run "journalctl --no-pager -b 0 > /tmp/full_journal.log";
```

Comment:
better not override the post_fail_hook from opensusebasetest but override, e.g. call

`$self->SUPER::post_fail_hook;`